Dan Collins blog post #100 (#100)
ippontech/blog-usa: posts/Ippon-USA-offers-a-helping-hand-to-those-impacted-by-natural-disasters.md

>  

Reviewer: The top of the page should have the following kind of information: https://github.com/ippontech/blog-usa#metadata

---
authors:
- Danny Collins
tags:
- life-at-ippon
- culture
date: 
title: "Ippon USA Offers a Helping Hand to Those Impacted By Natural Disaster"
image: 
---

>  

Reviewer: 1. I suggested the two tags [life-at-ippon, culture] because I thought they fit the best. Feel free to add your own.
2. The date can be blank.
3. You should find a main image for the blog, maybe something from your trip?

> Ippon USA offers a helping hand to those impacted by natural disasters. 

Reviewer: You don't need to include the title, it will be automatically added from the meta data.

 
“Let us remember: One book, one pen, one child, and one teacher, can change the world. Never doubt that a small group of thoughtful committed citizens can change the world. Indeed, it is the only thing that ever has.” - Malala Yousafzai
 

> About Me:

Reviewer: These subtitles should have a #'s for markdown. ie

# Level 1
## Level 2
### Level 3

I suggest making all your subtitles level 1: [About Me, Background, My Time On Project]

My name is Danny Collins and I work for Ippon Technologies USA as a senior project manager. I have 14 years experience in the field of change and project management across multiple sectors on several continents. 
Firstly, I am extremely fortunate to have a transferable skill set as a result of my career in project management, that can be used to build schools in rural Mexico as well as being able to manage a complex data migration in the abyss of the big data/cloud consulting world.

> Secondly, I am grateful to work for such a socially conscious and empathetic organisation as Ippon Technologies USA who have made my volunteering with All Hands & Hearts - Smart Response possible by allowing me the scope and flexibility to go off and help others, at the temporary expense of our deliverables here in the United States. Shout out to Romain Lheritier, our Managing Director and Jamie Bock, our Head of HR for not laughing me out of the room when I pitched this to them back in November. 

Reviewer 2: ```suggestion
Secondly, I am grateful to work for such a socially conscious and empathetic organization as Ippon Technologies USA who have made my volunteering with All Hands & Hearts - Smart Response possible by allowing me the scope and flexibility to go off and help others, at the temporary expense of our deliverables here in the United States. Shout out to Romain Lheritier, our Managing Director and Jamie Bock, our Head of HR for not laughing me out of the room when I pitched this to them back in November. 
```

Background:

> On September 7, 2017, one of the most lethal earthquakes in Mexican history hit close to the southern state of Chiapas, with a devastating magnitude of 8.2. Just 12 days later and a few hundred miles away, a 7.1 magnitude quake rocked central Mexico, toppling buildings, breaking gas mains, knocking out electricity and sparking fires across the city and other towns in central Mexico.

Reviewer: Consider adding images like maps, or pictures from your trip.
Images are added like so:

![Earthquake Map](https://link-to-image)

> More than a year after the devastating Earthquakes hit, many schools in Mexico remained unsafe for children to learn in, which forced many students to study in dangerous temporary learning centres. For this reason, we are continuing our work in Mexico within the Mexican state of Oaxaca.

Reviewer: Consider breaking up the last two paragraphs into smaller ones.

> More than a year after the devastating Earthquakes hit, many schools in Mexico remained unsafe for children to learn in, which forced many students to study in dangerous temporary learning centres. For this reason, we are continuing our work in Mexico within the Mexican state of Oaxaca.

Reviewer 2: ```suggestion
More than a year after the devastating Earthquakes hit, many schools in Mexico remained unsafe for children to learn in, which forced many students to study in dangerous temporary learning centers. For this reason, we are continuing our work in Mexico within the Mexican state of Oaxaca.
```

> The Oaxaca program began in January 2019, focusing on assisting the Celso Muñoz Primary School within the Oaxaca state, where our work will be centered around rebuilding 7 classrooms, retrofitting 4 classrooms and rebuilding the office and library to ensure that students have a safe and disaster-resilient place to learn. This school will allow 170 students, between the ages of 6 and 12 to return to school and receive the education they need for a brighter future. We look forward to working with this diverse and culturally-rich community to rebuild a school that will ensure they build a safer future for their students that will allow their diversity of culture to thrive for many years to come.

Reviewer 2: ```suggestion
The Oaxaca program began in January 2019, focusing on assisting the Celso Muñoz Primary School within the Oaxaca state, where our work includes rebuilding seven classrooms, retrofitting four classrooms and rebuilding the office and library to ensure that students have a safe and disaster-resilient place to learn. This school will allow 170 students, between the ages of 6 and 12 to return to school and receive the education they need for a brighter future. We look forward to working with this diverse and culturally-rich community to rebuild a school that will ensure they build a safer future for their students that will allow their diversity of culture to thrive for many years to come.
```

My Time on Project:

> 15 hours after setting off from my apartment in NYC and taking nearly every mode of transport available, I arrived at the HQ of the volunteer program in Oaxaca, Mexico. The so-called base is located in small neighbourhood La Santita, a barrio of Salinas Del Marquez. This town is separated by a massive harbour from the more well known Salina Cruz, famous for the oil refinery. 

Reviewer 2: ```suggestion
Fifteen hours after setting off from my apartment in NYC and taking nearly every mode of transport available, I arrived at the HQ of the volunteer program in Oaxaca, Mexico. The so-called base is located in a small neighborhood La Santita, a barrio of Salinas Del Marquez. A large harbor separates the town from the more well known Salina Cruz, famous for the oil refinery. 
```

> The next morning, after waking up early, the whole group of volunteers headed to the work site at 6.40 am. Currently, the group consist of approximately 40 people of all different ages from all over the world. All in purple, work boots on, sunscreen, water bottle and with a positive attitude: off to ‘work’. The volunteers make 8 long hour days, 6 days a week, in the extreme heat that marks this coastal area. Every now and then, they are blessed with a breeze. The region is known for high winds, but this year they are very absent, unfortunately. 

Reviewer 2: ```suggestion
The next morning, after waking up early, the whole group of volunteers headed to the work site at 6.40 am. Currently, the group consists of approximately 40 people of all different ages from all over the world. All in purple, work boots on, sunscreen, water bottle and with a positive attitude: off to ‘work.’ The volunteers make eight long hour days, six days a week, in the extreme heat that marks this coastal area. Now and then, they are blessed with a breeze. The region is known for high winds, but this year they are very absent, unfortunately. 
```

> On site, the group of volunteers works closely together with 15 masons, local labourers hired by the NGO. This comes with some difficulties from time to time because of the language barriers, just one of them speaks English and majority of the volunteers do not speak Spanish. Mostly, they work it out with sign language and the couple of words they know and it turns into a hilarious situation. Once a week, after work, the masons and volunteers will have ‘Spanglish’ classes, to learn the basics of the other language. 

Reviewer 2: ```suggestion
On-site, the group of volunteers works closely together with 15 masons, local laborers hired by the NGO. There are some difficulties from time to time because of the language barriers; just one of them speaks English, and the majority of the volunteers do not speak Spanish. Mostly, they work it out with sign language and the couple of words they know, and it turns into a hilarious situation. Once a week, after work, the masons and volunteers will have ‘Spanglish’ classes, to learn the basics of the other language. 
```

> The people living in Salinas del Marquez are very generous, open and loving. The masons invite you to their houses, or take you to one of the local bars, in someone’s garden. I was, and still am, humbled by the generosity from people that live in totally different circumstances than what I am used to. I have met so many amazing people and have been lucky to hear their great stories. The kids are screaming for us when we encounter them on the streets, they are happy that we are there to rebuild their school. This connection with communities which we would otherwise miss in our daily lives is part of the reason i keep coming back to serve with this organisation. I caught the bug after joining them on project in Nepal 2016, initially a 4 week commitment which quickly escalated into a 5 month tenure on project to complete the school we were building there. I have since answered the call to serve on both international and domestic projects, both within the US and its territories, including Puerto Rico where I joined the cleanup following hurricane Maria. 

Reviewer 2: ```suggestion
The people living in Salinas del Marquez are very generous, open and loving. The masons invite you to their houses, or take you to one of the local bars, in someone’s garden. I was and still am humbled by the generosity of people that live in totally different circumstances than what I am used to. I have met so many amazing people and have been lucky to hear their great stories. The kids are screaming for us when we encounter them on the streets, and they are happy that we are there to rebuild their school. 

This connection with communities which we would otherwise miss in our daily lives is part of the reason I keep coming back to serve with this organization. I caught the bug after joining them on a project in Nepal 2016, initially a 4-week commitment which quickly escalated into a 5-month tenure on a project to complete the school we were building there. I have since answered the call to serve on both international and domestic projects, both within the US and its territories, including Puerto Rico where I joined the cleanup following hurricane Maria. 
```

> On my last Sunday on project, we played football (soccer) against the local workers (known as masons) from the community. It was an early morning kick off but already extremely hot. We kicked off later than planned as both masons and volunteers were nursing hangovers from drinking together until late the evening before. It was an exhausting yet hilarious match which went into extra time and then happily, the volunteer team eventually won 3-2 on penalties. This game served to further create a closer connection between the international volunteers and the wider local community, as much like on the construction site, the language barrier fails to stop bonds and friendships being forged. This has since become a monthly fixture and will be continued for the duration of the project until late summer 2019.

Reviewer 2: ```suggestion
On my last Sunday on the project, we played football (soccer) against the local workers (known as masons) from the community. It was an early morning kick-off but already extremely hot. We kicked off later than planned as both masons and volunteers were nursing hangovers from drinking together until late the evening before.

It was an exhausting yet hilarious match which went into extra time and then happily, the volunteer team eventually won 3-2 on penalties. This game served to further create a closer connection between the international volunteers and the wider local community.  As on the construction site, the language barrier fails to stop bonds and friendships forged. This has since become a monthly fixture and will be continued for the duration of the project until late summer 2019.
```

> All in all, it has once again been a broad, culturally diverse experience, in a very welcoming community. The positive vibes and energy coming from the volunteers and all the people of the local community is amazing. They make you realize what you are there for: it is all for the kids. Rebuilding their school, so they can return to a safe learning environment to grow and have all the opportunities we take for granted. 

Reviewer: Consider making this another section like "Reflections" or "How You Can Help" (call to action)

> All in all, it has once again been a broad, culturally diverse experience, in a very welcoming community. The positive vibes and energy coming from the volunteers and all the people of the local community is amazing. They make you realize what you are there for: it is all for the kids. Rebuilding their school, so they can return to a safe learning environment to grow and have all the opportunities we take for granted. 

Reviewer 2: ```suggestion
All in all, it has once again been a broad, culturally diverse experience, in a very welcoming community. The positive vibes and energy coming from the volunteers and all the people of the local community are amazing. They make you realize what you are there for: it is all for the kids. Rebuilding their school, so they can return to a safe learning environment to grow and have all the opportunities we take for granted. 
```

It has once again been my great privilege to find the time and support to be able to serve alongside my fellow volunteers, under the banner of All Hands & Hearts, in the goal of helping those impacted by natural disaster who are unable to help themselves. 
If you too want to challenge the notion, that one person cannot change the world, just click the link below and go find out for yourselves. I’m certain you won’t regret it. 

> https://www.allhandsandhearts.org/volunteer/

Reviewer: I believe links should use [https://www.allhandsandhearts.org/volunteer/](https://www.allhandsandhearts.org/volunteer/) format. I may be wrong if it you just want the link to display.
